main/src/cgeo/geocaching/connector/gc/GCMap.java

```diff
@@ -187,6 +188,10 @@ private static void fillForBasicFilter(@NonNull final BaseGeocacheFilter basicFi
                 fillForBasicFilter(((DifficultyAndTerrainGeocacheFilter) basicFilter).difficultyGeocacheFilter, search);
                 fillForBasicFilter(((DifficultyAndTerrainGeocacheFilter) basicFilter).terrainGeocacheFilter, search);
                 break;
+            case RATING:
+                // not supported for online searches
+                // search.setRating(((RatingGeocacheFilter) basicFilter).getMinRangeValue(), ((RatingGeocacheFilter) basicFilter).getMaxRangeValue());
+                break;
```

Comment:
Ah this is what you meant? Yes, you can leave RATING out in the gcconnector.
But add it to Okapiclient (and also to the list of filters supported by it in implementation of `ISearchByFilter`)

Reply:
Thanks, have added it